src/benchmark/executor.rs

```diff
@@ -36,11 +38,13 @@ pub trait Executor {
 fn run_command_and_measure_common(
     mut command: std::process::Command,
     command_failure_action: CmdFailureAction,
+    command_input_policy: &CommandInputPolicy,
     command_output_policy: &CommandOutputPolicy,
     command_name: &str,
 ) -> Result<TimerResult> {
     let (stdout, stderr) = command_output_policy.get_stdout_stderr()?;
-    command.stdin(Stdio::null()).stdout(stdout).stderr(stderr);
+    let stdin = command_input_policy.get_stdin()?;
```

Comment:
move this above the `stdout, sterr` line to be consistent with the ordering elsewhere (input before output)?

Reply:
That has not been resolved yet.